docs/release-notes.md

```diff
@@ -1,5 +1,39 @@
 # Release notes
 
+## Trino Gateway 10 (24 July 2024)
+
+[JAR file gateway-ha-10-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/10/gateway-ha-10-jar-with-dependencies.jar),
+Docker container `trinodb/trino-gateway:10`
+
+* [:warning: Breaking change:](#breaking) Remove support for Dropwizard and
+  Jetty Proxy integration and usage. Add
+  [Airlift](https://github.com/airlift/airlift) as the base application
+  framework as
```

Comment:
NIT: the line break, not sure how it would look?

Reply:
This is an implementation detail so I'm not sure whatever we should call it out. Configuration changes obviously but not framework itself

Reply:
I think it is important since it affects custom module writing and other aspects quite a lot as well and our audience is currently not just administrators running Trino Gateway but also heavily developers modifying it.

Reply:
Rendering works fine.